pkg/utilities/manifest/manifest.go

```diff
@@ -38,14 +38,14 @@ var AddToScheme = SchemeBuilder.AddToScheme
 
 func addKnownTypes(scheme *runtime.Scheme) error {
```

Comment:
This function seems weird. I'd use the AddToScheme methods instead to get what's actually needed without duplicating code.